packages/mux-player/test/player.test.js

```diff
@@ -476,23 +476,22 @@ describe('seek to live behaviors', function () {
     await waitUntil(() => playerEl.inLiveWindow, 'clicking seek to live did not seek to live window');
   });
 
-  it.skip('should seek to live when play button is pressed', async function () {
+  it('should seek to live when play button is pressed', async function () {
     const playerEl = await fixture(`<mux-player
       playback-id="v69RSHhFelSm4701snP22dYz2jICy4E4FUyk02rW4gxRM"
       muted
       stream-type="ll-live"
       preload="auto"
     ></mux-player>`);
 
-    const mcPlayEl = playerEl.theme.shadowRoot.querySelector('media-play-button');
-    // NOTE: Need try catch due to bug in play+autoplay behavior (CJP)
-    try {
```

Comment:
I'm not seeing the need for this try/catch, but will re-add if it fails in CI.

Reply:
cool may have been resolved or at least side-stepped for the permutation with the preload+autoplay updates.